alerts/cluster-monitoring-operator/KubeAPIDown.md

```diff
@@ -7,29 +7,32 @@ been reachable by the monitoring system for more than 15 minutes.
 
 ## Impact
 
-This is a critical alert. The Kubernetes API is not responding. The
-cluster may partially or fully non-functional.
+This is a critical alert. It indicates that the Kubernetes API is not
+responding, and the cluster might be partially or fully non-functional.
 
 ## Diagnosis
 
-Check the status of the API server targets in the Prometheus UI.
+1. Verify the status of the API server targets in Prometheus in the OpenShift
+web console.
 
-Then, confirm whether the API is also unresponsive for you:
+1. Confirm whether the API is also unresponsive:
 
 ```console
 $ oc cluster-info
 ```
 
```

Comment:
Maybe try removing this newline here to fix the item count of the list?

Reply:
Thanks, @jan--f! Had to indent each line of sample code with 4 spaces. That seems to have fixed things. Learn something new every day!